### ISSUE TITLE
Unit Tests missing

### ISSUE THREAD

**Issue by ArminSchenk** [citoverse/cito#91]
Es fehlen noch tests (dnn/cnn) fuer loss functions "mvp" und "clogit"

**Comment by MaximilianPi** [comment]
Including cross-entropy, please check all loss functions, also the table in the documentation should state which input type (for the response) we allow (integer/character/factor)
